src/assets/locales/en/cookie-consent-translations.json

```diff
@@ -44,6 +66,11 @@
           ]
         }
       },
+      {
+        "title": "Social Embeds",
+        "description": "Social embeds are content from social networks (Facebook, Instagram, X, YouTube, and LinkedIn) that we integrate into our website so you can view the content without leaving the site. This process is known as 'embedding'. You can enable social embeds through this cookie consent solution now or later when you wish to view a social embed on the website. </br></br> Google is responsible for the YouTube platform under data protection law. Google uses cookies and similar technologies so that you can view YouTube content on this website. These technologies can only be used by Google after you have activated social embeds. Until then, you will remain invisible to Google.</br>This service is used on this website. </br></br> Company which processes the data </br>Google Ireland Limited, Gordon House, Barrow Street, Dublin 4, D04E5W5, Ireland. However, personal data may also be processed by Google LLC, 1600 Amphitheater Parkway, Mountain View, CA 94043, USA (data transfer mechanism: EU-US Data Protection Framework certification; EU standard contract).</br></br>Collected data </br>This list represents all (personal) data that is collected by or through the use of this service. </br> To our knowledge, Google processes your IP address and information about which social embed you have activated on our site. Further information on the processing of your personal data by Google can be found in the Google privacy policy at: <a href=\"https://policies.google.com/privacy?hl=en\" target=\"_blank\">https://policies.google.com/privacy?hl=en</a>. </br></br> Read the privacy policy of the data processor: <a href=\"https://policies.google.com/privacy?hl=en\" target=\"_blank\">https://policies.google.com/privacy?hl=en</a>",
```

Comment:
I would limit this description to YouTube and don't mention Facebook, Instagram, X, YouTube, and LinkedIn
```suggestion
        "description": "YouTube embeds are video content from YouTube that we integrate into our website, allowing you to view the content directly on our site without needing to navigate to YouTube. This process is known as “embedding.” You can enable YouTube embeds through this cookie consent solution now or later when you decide to view a YouTube embed on our website. </br></br> Google is responsible for the YouTube platform under data protection law. Google uses cookies and similar technologies so that you can view YouTube content on this website. These technologies can only be used by Google after you have activated social embeds. Until then, you will remain invisible to Google.</br>This service is used on this website. </br></br> Company which processes the data </br>Google Ireland Limited, Gordon House, Barrow Street, Dublin 4, D04E5W5, Ireland. However, personal data may also be processed by Google LLC, 1600 Amphitheater Parkway, Mountain View, CA 94043, USA (data transfer mechanism: EU-US Data Protection Framework certification; EU standard contract).</br></br>Collected data </br>This list represents all (personal) data that is collected by or through the use of this service. </br> To our knowledge, Google processes your IP address and information about which social embed you have activated on our site. Further information on the processing of your personal data by Google can be found in the Google privacy policy at: <a href=\"https://policies.google.com/privacy?hl=en\" target=\"_blank\">https://policies.google.com/privacy?hl=en</a>. </br></br> Read the privacy policy of the data processor: <a href=\"https://policies.google.com/privacy?hl=en\" target=\"_blank\">https://policies.google.com/privacy?hl=en</a>",
```